### PR TITLE
Story/claudia 6304 migrate templates

### PR DESCRIPTION
**Pull request by hmunfru**
#####  REVIEWER

@jesuspg
#### DESCRIPTION

Script for migrating PaaS Manager template into Murano
#### Test

done

### CODE REVIEW

File: package-generator/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+2) Activate the virtualenv with *source ENV/bin/activate*
+3) Install the requirements running *pip install -r requirements.txt
+   --allow-all-external*
+
+Now the system is ready to use. For future sessions, only the step2 is required.
+
+
+## Running
+### Generating Murano packages
+The script package-generator.py script is used for generating Murano Packages from product uploaded in PaaS Manager and SDC. In addition, once the packages have been
```
Comment (flopezag) [inline package-generator/README.md]: The script package-generator.py script... script twice..

File: package-generator/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+generated, a new Pull Request is done to the fiware-enabler github repository with the new changes
+
+ The following lines are shown by using the -h option
+    usage: package-generator.py [-h] -u USER -p PASSWORD -t TENANT_ID
+                            [-r REGION_NAME] [-k AUTH_URL] [-g UPLOAD]
+                            [-ug USER_GITHUB] [-pg PASSWORD_GITHUB]
+
+    Testing product installation using paasmanager
```
Comment (flopezag) [inline package-generator/README.md]: Testing product installation using paasmanager?

We should not test them with Murano?

File: package-generator/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    -p PASSWORD, --os-password PASSWORD
+                        valid password for a keystone
+    -t TENANT_ID, --os-tenant-id TENANT_ID
+                        user tenant_id for a keystone
+    -r REGION_NAME, --os-region-name REGION_NAME
+                        the name of region for a keystone
+    -k AUTH_URL, --os-auth-url AUTH_URL
+                        url to keystone <host or ip>:<port>/v2.0
+    -g UPLOAD, --os-upload UPLOAD
+                        To upload to github?
```
Comment (flopezag) [inline package-generator/README.md]: why question mark?

File: package-generator/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    -ug USER_GITHUB, --os-user_github USER_GITHUB
+                        user github
```
Comment (flopezag) [inline package-generator/README.md]: github user

File: package-generator/README.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    -pg PASSWORD_GITHUB, --os-password_github PASSWORD_GITHUB
+                        password github
```
Comment (flopezag) [inline package-generator/README.md]: github password

File: package-generator/migrate-templates.py [hunk 6]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2014 Telefónica Investigación y Desarrollo, S.A.U
```
Comment (flopezag) [inline package-generator/migrate-templates.py]: 2014? or 2014-2016

File: package-generator/migrate-templates.py [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    migrate_templates(auth_url=args.auth_url,
+                      tenant_id=args.tenant_id,
+                      user=args.user,
+                      password=args.password,
+                      region_name=args.region_name)
+
+
+def migrate_templates(auth_url, tenant_id, user, password, region_name):
+    """
+    It migrate the abstract templates from PaaS Manager to Murano
```
Comment (flopezag) [inline package-generator/migrate-templates.py]: migrates

File: package-generator/model/tier.py [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+        :return: nothing
+        """
+        self.tier_name = tier_name
+        self.flavour = flavour
+        self.image = image
+        self.products = products
+
+    def add_products_from_xml(self, productReleaseDtos, template_id):
+        """
+        It add products to the tier from XML file.
```
Comment (flopezag) [inline package-generator/model/tier.py]: adds

File: package-generator/other.sh [hunk 9]
```diff
@@ -1 +1,2 @@
 python package-generator.py -u admin -g False -t 00000000000003228460960090160000 -p XXX
```
Comment (flopezag) [inline package-generator/other.sh]: why do not use $1, $2 and maybe $3 for user, password and tenant id and pass these parameters in the CLI?

File: package-generator/util/configuration.py [hunk 10]
```diff
@@ -90,3 +92,14 @@ def get_all_nids(self):
             all_nids.update(nids)
             params[chapter] = False
         return all_nids
+
+    def list_packages_murano(self):
+        """
+        It obtains all package from Murano
+        :return: the list with the murano packages.
+        """
+        array = []
```
Comment (flopezag) [inline package-generator/util/configuration.py]: I prefer to use:

array = list()

File: package-generator/README.md (hunk 5 above)
Comment (pratid) [reply package-generator/README.md]: Best practices suggest using one single letter for "short" options and --dash-separated-list-of-words for the "long" ones. If needed, the uppercase letters may resolve conflicts .

**Comment by flopezag** [comment]
LGTM

**Pull request merged**